src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests.csproj

```diff
@@ -55,4 +57,7 @@
       <Link>Common\CoreLib\System\PasteArguments.Unix.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup>
+    <SupplementalTestData Include="$(RootOutputPath)\native\$(BuildConfiguration)\SleepTestAppNative*" />
```

Comment:
@ViktorHofer is this the most appropriate way to hook this up? It's the first time a test has had a native SupplementalTestDAta. (Some process tests have native test data but they are Windows only vcxproj.)